### PR TITLE
Adding in Discreet to Duration conversion.

### PR DESCRIPTION
**Pull request by sublet**
Duration needs to be calculated based on points data.  Basically, going into the database, we need to group points where the LAT/LNG doesn't change over a given time and coming out it reverts the operation.

### CODE REVIEW

File: test/integration/upload.test.js [hunk 1]
```diff
@@ -137,7 +137,7 @@ describe('POST /case/points', () => {
         accessCode: currentAccessCode.value,
         caseId: currentCase.caseId,
       });
-      
+      console.log(result.error)
```
Comment (MarshallMcCoy) [inline test/integration/upload.test.js]: Probably want to delete this before we merge?

File: db/models/points.js [hunk 2]
```diff
@@ -85,6 +86,7 @@ class Service extends BaseService {
       record.hash = hash.encodedString
       record.coordinates = this.makeCoordinate(point.longitude, point.latitude);
       record.time = new Date(point.time);
+      record.duration = point.duration;
```
Comment (MarshallMcCoy) [inline db/models/points.js:89]: We need to make sure we're capturing `duration` up in `createRedactedPoint` as well.

File: db/models/cases.js [hunk 3]
```diff
@@ -199,7 +199,8 @@ class Service extends BaseService {
                 'points.id AS pointId',
                 'points.coordinates',
                 'points.time',
-                'points.hash'
+                'points.hash',
```
[the remark this answers is not in the document]
Comment (sublet) [reply db/models/cases.js:202]: It doesn't...removed it.

File: app/lib/pocTransform.js [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+
+const durationPointToDiscreetPoints = durationPoint => {
+  const discreetPoints = [];
+  let i;
+  for (i = 0; i < durationPoint.duration / 5; i++) {
+    discreetPoints[discreetPoints.length] = {
+      latitude: durationPoint.latitude,
+      longitude: durationPoint.longitude,
+      time: durationPoint.time + i * 5 * MINUTE,
+      hash: durationPoint.hash,
```
[the remark this answers is not in the document]
Comment (sublet) [reply app/lib/pocTransform.js:128]: Hashing is now done after we go from duration to discrete.

**Pull request merged**